### PR TITLE
adding boris_spassky, jungle_drum_bass files

### PR DESCRIPTION
**Pull request by HadeerAIA**
Please check this files

### CODE REVIEW

**Review by jbampton — CHANGES_REQUESTED** [review]
Please use all lowercase filenames

**Comment by HadeerAIA** [comment]
> Please use all lowercase filenames
It's only the title of the pull request but the files inside all in lowercase, but I will fix it too

**Review by jbampton — CHANGES_REQUESTED** [review]
So the actual files themselves like the picture of Boris should be all lowercase as well

**Review by jbampton — CHANGES_REQUESTED** [review]
Page should be in the "Sound" section

File: doc/source/jungle_drum_bass.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Sources
+
+-	MENNYEK KAPUI - Az elektroniks zene évtizede (The decade of electronic music)
+
+-	Wikipedia
+
+-	Ishkur's Guide to Electronic Music
+
+-	Every Noise at Once
```
Comment (jbampton) [inline doc/source/jungle_drum_bass.md:31]:
```suggestion
- MENNYEK KAPUI - Az elektroniks zene évtizede (The decade of electronic music)
- Wikipedia
- Ishkur's Guide to Electronic Music
- Every Noise at Once 
```

**Pull request merged**